agnpy/wrappers/gammapy.py

```diff
@@ -0,0 +1,208 @@
+from multiprocessing.sharedctypes import Value
+from xml.etree.ElementInclude import XINCLUDE_INCLUDE
+import astropy.units as u
+from ..synchrotron import Synchrotron
+from ..compton import ExternalCompton
+from ..compton import SynchrotronSelfCompton
+from gammapy.modeling.models import SpectralModel, Parameter, Parameters
+
+
+class SpectralModelSSC(SpectralModel):
```

Comment:
Maybe the wrapped models can follow the naming conventions of gammapy models with SpectralModel at the end :
`SynchrotronSelfComptonSpectralModel`, `ExternalComptonSpectralModel`

Reply:
Done, renamed following the `Gammapy` naming scheme.